KidsIdKit.Web/jest.config.js

```diff
@@ -0,0 +1,30 @@
+﻿// jest.config.js
```

Comment:
I don't know the answer to this - but it seems like this file should be in the wwwroot folder?

Reply:
For what it's worth, these are exclusively used for the test suite, so not anything that wwwroot would require. If desired, I could see how reworking the directory tree would look.

Reply:
Don't worry about it - if they are for the tests that is fine.